<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyZip</string>
  </property>
  <property name="windowIcon">
   <iconset resource="menu.qrc">
    <normaloff>:/file/images/myzip.png</normaloff>:/file/images/myzip.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>48</width>
    <height>48</height>
   </size>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>选择文件或文件夹</string>
    </property>
    <widget class="QComboBox" name="comboBox_fileToCtrl">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAtCurrent</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_find">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_file">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>89</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>文件</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_dir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>89</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>文件夹</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>541</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>191</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>详细信息</string>
     </property>
     <widget class="QTextBrowser" name="textBrowser_fileInfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>21</y>
        <width>171</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>20</width>
       <height>231</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>311</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>解压缩信息</string>
     </property>
     <widget class="QTextBrowser" name="textBrowser_compInfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>291</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>历史记录</string>
     </property>
     <widget class="QComboBox" name="comboBox_dir">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtCurrent</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rectrl">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>操作</string>
      </property>
     </widget>
    </widget>
    <zorder>groupBox_5</zorder>
    <zorder>groupBox_3</zorder>
    <zorder>line_2</zorder>
    <zorder>groupBox_4</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_E"/>
    <addaction name="separator"/>
    <addaction name="action_U"/>
    <addaction name="action_UT"/>
    <addaction name="separator"/>
    <addaction name="action_Q"/>
   </widget>
   <widget class="QMenu" name="menu_D">
    <property name="title">
     <string>编辑(&amp;D)</string>
    </property>
    <addaction name="action_D"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>工具(&amp;O)</string>
    </property>
    <addaction name="action_V"/>
    <addaction name="separator"/>
    <addaction name="action_H"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_B"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_D"/>
   <addaction name="menu"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_E"/>
   <addaction name="separator"/>
   <addaction name="action_U"/>
   <addaction name="action_UT"/>
   <addaction name="separator"/>
   <addaction name="action_V"/>
   <addaction name="separator"/>
   <addaction name="action_D"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_E">
   <property name="icon">
    <iconset resource="menu.qrc">
     <normaloff>:/file/images/com.png</normaloff>:/file/images/com.png</iconset>
   </property>
   <property name="text">
    <string>压缩(&amp;E)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_U">
   <property name="icon">
    <iconset resource="menu.qrc">
     <normaloff>:/file/images/uncom.png</normaloff>:/file/images/uncom.png</iconset>
   </property>
   <property name="text">
    <string>解压(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_UT">
   <property name="icon">
    <iconset resource="menu.qrc">
     <normaloff>:/file/images/uncomt.png</normaloff>:/file/images/uncomt.png</iconset>
   </property>
   <property name="text">
    <string>解压到</string>
   </property>
  </action>
  <action name="action_Q">
   <property name="icon">
    <iconset resource="menu.qrc">
     <normaloff>:/file/images/quit.png</normaloff>:/file/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_D">
   <property name="icon">
    <iconset resource="menu.qrc">
     <normaloff>:/file/images/del.png</normaloff>:/file/images/del.png</iconset>
   </property>
   <property name="text">
    <string>删除(&amp;D)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_V">
   <property name="icon">
    <iconset resource="menu.qrc">
     <normaloff>:/file/images/info.png</normaloff>:/file/images/info.png</iconset>
   </property>
   <property name="text">
    <string>查看属性(&amp;V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_H">
   <property name="icon">
    <iconset resource="menu.qrc">
     <normaloff>:/file/images/history.png</normaloff>:/file/images/history.png</iconset>
   </property>
   <property name="text">
    <string>浏览历史(&amp;H)</string>
   </property>
  </action>
  <action name="action_B">
   <property name="icon">
    <iconset resource="menu.qrc">
     <normaloff>:/file/images/about.png</normaloff>:/file/images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;B)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="menu.qrc"/>
 </resources>
 <connections/>
</ui>
